bigquery.ts

```diff
@@ -198,30 +206,43 @@ const sendSearchQuery: SendSearchQuerySignature = async function(searchParams) {
   const queries = [
     bigQuery(query, { keywords, excludedKeywords, locale, taxon, organisation, link })
   ];
-  if (selectedWordsWithoutQuotes &&
-    selectedWordsWithoutQuotes.length > 5 &&
-    selectedWordsWithoutQuotes.includes(' ')) {
-    queries.push(bigQuery(
-    `SELECT *
-     FROM search.thing
-     WHERE CONTAINS_SUBSTR(name, @selected_words_without_quotes)
-     ;`
-    , { selectedWordsWithoutQuotes }))
-  }
-
-
-  const results = await Promise.all(queries);
-
-  const bqMainResults = results[0];
-  const bqMetaResults = results.length > 1 ? results[1] : [];
 
+  let bqMetaResults: MetaResult[] = [], results: any, bqMainResults: MainResult[] = [];
+  switch (searchParams.searchType) {
+    case SearchType.Taxon:
+      console.log('SearchType taxon');
+      results = await Promise.all(queries);
+      bqMainResults = results[0];
```

Comment:
These lines can come before the `switch`, similarly lines 220-221

Reply:
No, because of the `default:` section where the meta search is added to `queries`

Reply:
Can they come after, then?

Reply:
Not sure, because the `results` is set differently in the `default` clause.
I'm aware that this could be written better though.